### PR TITLE
Add bps 3.6.0 release. features of v4.4.10

### PR DESCRIPTION
**Pull request by milindaperera**

### CODE REVIEW

**Review by thusithathilina — COMMENTED** [review]
I checked few random features in the BPS 3.6.0 against the PR seems some features are missing in the PR.
e.g. org.wso2.bps:org.wso2.bps.processtools is not there in the PR but exist in the product p2.
Could you please check if all the features which packed in the product contains in the PR

**Comment by milindaperera** [comment]
added org.wso2.bps.multitenancy.dashboard.ui.feature, other features are not needed they contain util jars for samples, and process clean up tool etc.

File: pom.xml [hunk 1]
```diff
@@ -266,6 +267,7 @@
         <product.cep.version.4.0.0>4.0.0</product.cep.version.4.0.0>
         <product.cep.version.4.1.0>4.1.0</product.cep.version.4.1.0>
         <product.esb.version.4.9.0>4.9.0</product.esb.version.4.9.0>
+        <product.bps.version.3.6.0>3.6.0</product.bps.version.3.6.0>
```
Comment (thusithathilina) [inline pom.xml]: This need to be in alphabetical order

Comment (milindaperera) [reply pom.xml]: fixed in the commit 3459f21

**Pull request merged**